### ISSUE TITLE
Some alternative package options for consideration

### ISSUE THREAD

**Issue by jcmkk3** [lambdaisland/corgi#5]
Hello,

First, I want to say that your approach here really resonates with me. It feels very selective and light to the touch compared to Spacemacs or Doom, while being a great basis to customize. To that end, I'd like to throw out a couple of packages that are lighter weight than your current choices and might be interesting to you. If you've already considered them, then no problem.

For completion frameworks, [Selectrum](https://github.com/raxod502/selectrum) or [Icomplete Vertical](https://github.com/oantolin/icomplete-vertical), along with  [prescient](https://github.com/raxod502/prescient.el) or [orderless](https://github.com/oantolin/orderless).

[Here](https://github.com/raxod502/selectrum#ivy) is a comparison of Selectrum vs Ivy. Icomplete Vertical shares a lot of the same advantages, but is even more modest in scope.

There's also a [whole section](https://github.com/raxod502/selectrum#complementary-extensions) about all of the extensions that are available to extend either of these options, since they're both based on the built-in completing-read. This ecosystem of packages are great because they're all usable with vanilla Emacs outside of any particular completion framework.

You also may be able to replace projectile with the built in [project.el](http://elpa.gnu.org/packages/project.html). Manuel Uberti wrote a [couple](https://www.manueluberti.eu/emacs/2020/09/18/project/) [posts](https://www.manueluberti.eu/emacs/2020/11/14/extending-project/) about it recently.

**Comment by plexus** [comment]
Hi @jcmkk3, thanks for checking out Corgi. I was not aware of Selectrum, I just went through the README and it looks quite compelling. I'll be trying it out.

I'm a big fan of straight.el, I would not be able to do Corgi if it wasn't for straight, so I'm quite happy to put my faith in @raxod502.

Meanwhile quite a bit has changed, Corgi is no longer a config as such, it's now just a set of packages that can be pulled in with straight. This was the last big fundamental change I still wanted to make, from here things should be more iterative. I've also started work on some proper documentation, still early stage but it does show how you're expected to install and use Corgi going forward: https://github.com/plexus/corgi/blob/main/corgi_manual.org

**Comment by jcmkk3** [comment]
Great. I tried the configuration example and it worked great except for a couple of the optional use-package suggestions. I had to change them to `markdown-mode` and `yaml-mode`.

Also, for the Emacs installation on Windows, I would suggest to use [scoop](https://scoop.sh/). It is like homebrew for Windows. It does not require admin privileges, so it is great for corporate computers. It has pretty much all useful developer cli tools and languages. To install emacs, the user can follow instructions on [this issue](https://github.com/lukesampson/scoop/issues/335).

A clojurian bucket can be found [here](https://github.com/littleli/scoop-clojure)
